os/kernel/binary_manager/binary_manager_bootparam.c

```diff
@@ -268,22 +269,28 @@ void binary_manager_update_bootparam(int requester_pid, uint8_t type)
 		return;
 	}
 
+	response_msg.result = BINMGR_OK;
+	is_all_updatable = true;
+
 	/* Get current bootparam data and update version */
 	memcpy(&update_bp_data, binary_manager_get_bpdata(), sizeof(binmgr_bpdata_t));
 	update_bp_data.version++;
```

Comment:
Regardless of this commit, bp version should be the last of update sequence. If not, when power off is happened at middle of the update, bootloader can check wrong bp first.

Reply:
This value is a local variable and not written in flash until bootparam checking is done.
So I think this is not problem.